.gitignore

```diff
@@ -1,4 +1,11 @@
 .DS_Store
 node_modules/
 .tmp/
-
+.vscode/
```

Comment:
I included these files as part of the repo for developer productivity by using a common development environment and workflows, eg. if you clone the repo, your development environment is installed. Could you explain why you would propose to remove them?